source/managing-teams/holding-one-on-ones.html.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Holding 1:1s
+meta_description: >
+  1:1s are a ritual that allows Engineering Managers and their teams to talk about well-being,
+  professional growth and career progression.
+---
+
+All Engineering Managers are expected to have recurring 1:1 meetings (one-on-ones) with their team.
+These meetings are typically held every two weeks and they are 30-minutes long, although the
+frequency and length are decided with the report. Both parties should equally contribute to the
+agenda, which may or may not have fixed sections and discussion points.
+
+Thanks to their flexibility, one-on-ones are possibly the most valuable tool at a manager's
+disposal. They create a fixed cadence through which you can carry out a lot of your
+responsibilities, such as:
+
+- Gauging the teammate's morale and overall satisfaction with Nebulab.
+- Supporting the teammate in their personal and professional growth.
+- Providing and receiving positive and constructive feedback.
+- Keeping your teammate in the loop about what's going on in the company.
+- Having performance and compensation-related conversations.
+
+No matter your workload and priorities, 1:1s should be considered a sacred ritual. It's okay to
+reschedule a 1:1 every once in a while if absolutely needed—it's NOT okay to skip them.
+
+## What makes a great 1:1
+
+When it comes to 1:1s, there is no right way to do them: every teammate has their communication
+style and preferences. The secret to a great 1:1 is understanding what your report is interested in
+discussing, and adapting your management style to it. At the same time, you don't want to come
+across as fake, so make sure you bring your whole self to the 1:1 as well.
+
+Here are some ground rules to follow for holding effective 1:1s:
+
+- **Prepare thoroughly.** Don't jump into a 1:1 with no idea of what you're going to discuss. Review
+  the agenda from the past meeting and use it to prepare the new one. Make sure to always ask your
+  report if they have any points _they_ would like to discuss!
+- **Show genuine interest.** Your main job during a 1:1 is to build trust and rapport, so that the
+  other person is comfortable telling you what's really on their mind, rather than what they think
+  you want to hear. The only way to build rapport is to show genuine interest in them.
+- **Build accountability.** The only thing worse than not holding 1:1s is holding useless 1:1s,
+  where it feels like you're both wasting your time. To make your 1:1s useful, build
+  accountability: have clear action items for each meeting and follow up on them as quickly as
+  possible.
+
+One more tip: it can be tempting to schedule all your 1:1s on the same day, so that the rest of your
+week is free to do other work. However, 1:1s are very demanding—it is hard to effectively do more
+than one or two a day. Instead, try to spread them out throughout the week, so that you can give
```

Comment:
I don't think `throughout the week` is 100% correct here. eg. suppose the 1-1 meetings are all monthly, it should be `throughout the month`. The occurrence can change of course so maybe `as much as possible` is better?